### PR TITLE
New package: sam-astro.Z-Sharp version 2.1.3

### LINKED ISSUE #156094: [Package Request]: Z-Sharp
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/sam-astro/Z-Sharp/releases
Publisher: https://github.com/sam-astro
Package Name: `ZSharp`
Description: `Custom programming interpreter for ZSharp (Z#)`
Package Version: `v2.1.3-alpha`
Installer URL: https://github.com/sam-astro/Z-Sharp/releases/download/v2.1.3/ZSharp-Win-Installer.zip **OR**  https://github.com/sam-astro/Z-Sharp/releases/download/v2.1.3/ZS_Win_Base_Raw.zip

### PR DESCRIPTION
**Pull request by Exorcism0666**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?
- Resolve #156094

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156189)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files (x86)\ZSharp\ZSharp.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 882.)

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: sam-astro.Z-Sharp
+PackageVersion: 2.1.3-alpha
```
Comment (stephengillie) [inline manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.installer.yaml:5]: Please update file path to match this line.

File: manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: sam-astro.Z-Sharp
+PackageVersion: 2.1.3-alpha
```
Comment (stephengillie) [inline manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.locale.en-US.yaml:5]: Please update file path to match this line.

File: manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: sam-astro.Z-Sharp
+PackageVersion: 2.1.3-alpha
```
Comment (stephengillie) [inline manifests/s/sam-astro/Z-Sharp/2.1.3/sam-astro.Z-Sharp.yaml:5]: Please update file path to match this line.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Automatic Validation ended with:
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\s\sam-astro\Z-Sharp\2.1.3', expected 
'manifests\s\sam-astro\Z-Sharp\2.1.3-alpha'

**Pull request closed without merging**